src/current/v24.1/known-limitations.md

```diff
@@ -6,699 +6,624 @@ keywords: gin, gin index, gin indexes, inverted index, inverted indexes, acceler
 
-### Table-level restore will not restore user-defined functions
+#### Using `default_int_size` session variable in batch of statements
 
-{% include {{ page.version.version }}/known-limitations/restore-udf.md %}
+When setting the `default_int_size` [session variable]({% link {{ page.version.version }}/set-vars.md %}) in a batch of statements such as `SET default_int_size='int4'; SELECT 1::IN`, the `default_int_size` variable will not take affect until the next statement. This happens because statement parsing takes place asynchronously from statement execution.
 
-### Incorrect query plans for partitions with `NULL` values
+As a workaround, set `default_int_size` via your database driver, or ensure that `SET default_int_size` is in its own statement. [Tracking GitHub issue](https://github.com/cockroachdb/cockroach/issues/32846)
 
-In cases where the partition definition includes a comparison with `NULL` and a query constraint, incorrect query plans are returned. However, this case uses non-standard partitioning which defines partitions which could never hold values, so it is not likely to occur in production environments.
+#### Overload resolution for collated strings
 
-[Tracking GitHub issue](https://github.com/cockroachdb/cockroach/issues/82774)
+Many string operations are not properly overloaded for [collated strings]({% link {{ page.version.version }}/collate.md %}), for example:
 
-### `null_ordered_last` does not produce correct results with tuples
+{% include_cached copy-clipboard.html %}
+~~~ sql
+> SELECT 'string1' || 'string2';
+~~~
 
-By default, CockroachDB orders `NULL`s before all other values. For compatibility with PostgreSQL, the `null_ordered_last` [session variable]({% link {{ page.version.version }}/set-vars.md %}) was added, which changes the default to order `NULL`s after all other values. This works in most cases, due to some transformations CockroachDB makes in the optimizer to add extra ordering columns. However, it is broken when the ordering column is a tuple.
+~~~
+     ?column?
+------------------
+  string1string2
+(1 row)
+~~~
 
-[Tracking GitHub issue](https://github.com/cockroachdb/cockroach/issues/93558)
+{% include_cached copy-clipboard.html %}
+~~~ sql
+> SELECT ('string1' collate en) || ('string2' collate en);
+~~~
 
-### Limitations for `DROP OWNED BY`
+~~~
+pq: unsupported binary operator: <collatedstring{en}> || <collatedstring{en}>
+~~~
 
-[`DROP OWNED BY`]({% link {{ page.version.version }}/drop-owned-by.md %}) drops all owned objects as well as any [grants]({% link {{ page.version.version }}/grant.md %}) on objects not owned by the [role]({% link {{ page.version.version }}/security-reference/authorization.md %}#roles).
+[Tracking GitHub issue](https://github.com/cockroachdb/cockroach/issues/10679)
```

Comment:
I'm not sure about these links and UX. In release notes, for example, we make the link text something like "#10679". Maybe "This is tracked in [#10679](https://github.com/cockroachdb/cockroach/issues/10679)"

Reply:
Yeah, I didn't think about this too hard; mostly took the existing convention and made it consistent. Can you say more about why the UX doesn't work for you?

Reply:
If I'm using a screenreader because I'm visually impaired, my screen reader will read the text "Tracking GitHub issue" and I want to know which one, I have to make it read me the entire URL too. At least if the screenreader reads "Tracked in issue #10679", I'm probably used to that kind of formulation from my exposure to GitHub and/or from reading CockroachDB release notes, where we use a similar convention. WDYT?

Reply:
Updated!